### ISSUE TITLE
PSSE import - transformer impedance scaling and winding voltage

### ISSUE THREAD

**Issue by KSepetanc** [lanl-ansi/PowerModels.jl#629]
I tested PSSE network on polar (exact) OPF and after debugging and thorough PSSE documentation reading I found out it was transformer impedance problem. To get `Zeq` from PSSE data, `Z` from PSSE should be multiplied by `|t2|^2=("WINDV2"/"NOMV2")^2`. After doing so, power flow matched exactly PSSE calculations. I think PowerModels should automatically scale data into correct `Zeq`. The change is rather significant (10% easily in `R` and `X`).

I found about this in PSSE documentation: 
```
Program Application Guide - Volume 1 (v 33.5);
PAGV1.pdf; 
page 97 (section 4-7); 
equation 4.21b
available in PSSE installation folder
```

Additionally, having separate |t1| and |t2| is beneficial when dealing with tap position optimizations. Thus, I think `"WINDV1"`, `"WINDV2"`, `"NOMV1"` and `"NOMV2"` should be available when `import_all=true`.

So I propose changing tap calculation in `psse.jl` so it does not delete `"WINDV1"` and `"WINDV2"`:
line 387         `sub_data["tap"] = transformer["WINDV1"] / transformer["WINDV2"]`
and removing:  `"NOMV1"` and `"NOMV2"` from `_import_remaining!`.

**Comment by ccoffrin** [comment]
@KSepetanc thanks for reporting this.  I don't have easy access to PSSE to verify solutions side-by-side, so I am not surprised that these kinds of inconsistencies exist.

I agree `import_all` should include all of the values, even the ones we translate into other fields.  So on that point a PR is welcome.

Concerning the more accurate calculation of `|t2|^2=("WINDV2"/"NOMV2")^2`.  I am a little hesitant to do this by default, becouse I am worried about the data quality across the wide variety of data sources that these networks come from. 

@pseudocubic, @frederikgeth any thoughts?

**Comment by KSepetanc** [comment]
@ccoffrin I will try to explain briefly why scaling is needed. Impedance `Z` is something that should be available from nameplates which assume a bit more detailed replacement scheme not suitable for OPF (due to more variables). PSSE and PowerModels use the same simplified scheme for internal calculations (besides parallel shunt that is not relevant for this conversation). The simplified scheme is equivalent to the original only if `Z` is scaled into `Zeq` as described in the first comment.

So, the scaling nameplates values is something that is always needed for PowerModels transformer model (if `|t2|!=1`). This in my opinion this raises a concern that PowerModels models based on pglib data are also off by some margin since I am not sure if pglib's `R` and `X` account for `|t2|`.

Regarding `import_all`, it is very trivial change in the code. Should I open commit? Can I change it directly online? I have changes already implemented locally since I needed the data. I loaded the modified package using `DEV` in Julia. Short instructions are welcome.

**Comment by KSepetanc** [comment]
Any progress on this topic?

**Comment by ccoffrin** [comment]
Updates are stalled while I resolve the issue with the tests relating to breaking changes in the latest version of the SCS.  If you would like to see these changes integrated sooner rather than later then please make to separate PRs one for the `import_all` fix and another for the proposed `|t2|` fix.

**Comment by KSepetanc** [comment]
@ccoffrin I noticed that some work has been done on PSSE transformers recently (#715) so we could fix this transformer issue as well.

Do we need a new branch to create pull request? I am not well fond of PR concept.

**Comment by ccoffrin** [comment]
Sure, it would be great to fix this.  If you could make a PR that would be great.  The process is to make a fork of this repo into your personal github space; you then make the changes the master branch of your copy of the repo (use the same Dev procedure you used before, but make sure to install your copy of the PowerModels package).  Once the changes are working, you can make a PR from your local copy back to this repository.

**Comment by ccoffrin** [comment]
closed by PR #722
